ex_topo/ex_topology_output.yaml

```diff
@@ -102,13 +102,13 @@ os_server_res:
     changed: true
     id: f5832fcc-4e1b-442d-8be8-ba5e3783e7f2
     instance:
-    - http://dashboard.centralci.eng.rdu2.redhat.com:5000/v2.0/
-    - e2e-openstack
-    - rgHdUfMqshXWSBYdlfIp
-    - e2e-openstack
+    - <endpoint>
+    - <username>
+    - <password>
+    - <username>
     - present
     - rhel-6.5_jeos
-    - ci-factory
+    - <ssh-key-reference>
     - m1.small
     - e2e-openstack
```

Comment:
@arilivigni so that change says that I removed the named reference and replaced it with <ssh-key-reference>. Is that not what you want?